### PR TITLE
doc(helm-guides): add example app installation

### PR DESCRIPTION
**Pull request by leiicamundi**
## Description

This PR extends the existing EKS guide by incorporating the demo payment example C8 application, developed by @ingorichtsmeier, which serves as a validation tool for the C8 stack. The objective is to showcase a working example within the EKS environment.

## When should this change go live?

- [x] This change is not yet live and should not be merged until https://github.com/camunda-community-hub/camunda-8-examples/pull/342 is merged
- [x] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [x] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Review by maxdanilov — CHANGES_REQUESTED** [review]
(Reviewing this PR instead of Lars)

I've left some comments on the wording. Additional thing missing in this PR, is that the new pages (in order to appear on the left sidebar in the table of contents) need to be added to https://github.com/camunda/camunda-docs/blob/main/sidebars.js and https://github.com/camunda/camunda-docs/blob/main/optimize_sidebars.js in the corresponding version branches.

**Comment by christinaausley** [comment]
@leiicamundi is this still in `draft` because you have additional changes coming? Let me know when you are ready for final review. We can pull this out of draft if there are no additional large changes, as the `hold` label is sufficient.

**Review by maxdanilov — CHANGES_REQUESTED** [review]
Content-wise LGTM, one missing thing is sidebars (they are versioned as well), in this case:
- https://github.com/camunda/camunda-docs/blob/main/versioned_sidebars/version-8.4-sidebars.json
- https://github.com/camunda/camunda-docs/blob/main/versioned_sidebars/version-8.5-sidebars.json
(should be updated because a new page is added)

**Review by maxdanilov — APPROVED** [review]

**Comment by christinaausley** [comment]
I'll get this reviewed today 👍

File: docs/self-managed/setup/guides/installing-payment-example.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+id: installing-payment-app-example
+title: "Verifying Camunda 8 installation with the payment example process application"
```
Comment (christinaausley) [inline docs/self-managed/setup/guides/installing-payment-example.md:3]: This title is a little lengthy for a sidebar. We can keep as-is, but maybe add a sidebar_label named something like "Verifying Camunda 8 installation"?

Comment (maxdanilov) [reply docs/self-managed/setup/guides/installing-payment-example.md:3]: What about `Verifying Camunda 8 installation with an demo app`?

**Pull request merged**